tests/ttnn/unit_tests/operations/eltwise/test_add.py

```diff
@@ -487,3 +488,30 @@ def test_add_with_block_sharding(device, input_a_sharded, input_b_sharded, out_s
     output_tensor = ttnn.to_torch(output_tensor)
     assert ttnn.pearson_correlation_coefficient(torch_output_tensor, output_tensor) >= 0.99988
     assert output_tensor.shape == shape
+
+
+@pytest.mark.parametrize(
+    "data",
+    [
+        ([], [], []),
+        ([1], [2], [3]),
+        ([1], [], []),
+        ([], [1], []),
+        ([1, 2], [3], [4, 5]),
+        ([1], [2, 3], [3, 4]),
+        ([1, 2], [3, 4], [4, 6]),
+    ],
+)
+@pytest.mark.parametrize("memory_config", [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG])
+def test_01_volume_tensors(device, data, memory_config):
+    (a, b, c_golden) = data
+    a = torch.BFloat16Tensor(a)
+    b = torch.BFloat16Tensor(b)
+    assert torch.add(a, b).tolist() == c_golden
```

Comment:
@sminakov-tt ,  don't we want to also support torch.zeros((0,1)) or torch.empty((0,1)) whereby the shape is (0,1) but the tensor is still treated like a scalar?

Reply:
@eyonland , shape [0, 1] is not a Scalar - thats an empty tensor, volume is 0

Reply:
```py
In [2]: import torch
In [3]: a = torch.ones(0,1)
In [4]: a
Out[4]: tensor([], size=(0, 1))
In [5]: b = torch.ones(1,1)
In [6]: c = a + b
In [7]: c
Out[7]: tensor([], size=(0, 1))
In [9]: a
Out[9]: tensor([], size=(0, 1))
In [10]: d = b / a
In [11]: d
Out[11]: tensor([], size=(0, 1))
```

Reply:
```
In [1]: import torch
In [2]: a = torch.zeros(0, 1)
In [3]: b = torch.ones(1,1)
In [4]: c = a + b
In [5]: c
Out[5]: tensor([], size=(0, 1))
```